### ISSUE TITLE
bug(mat-select): Talkback cannot navigate long dropdowns

### ISSUE THREAD

**Issue by shanti-razr** [angular/components#32385]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

When navigating through a mat-select element using Talkback on an Android mobile device, dropdowns with many options cause the page to not maintain focus and result in major jittering and page UI issues. This is not the case with a list of small options. This may be partially a mat-option issue as well.

### Reproduction

StackBlitz link: https://stackblitz.com/edit/sb1-zewscnst?file=.env
Steps to reproduce:
1. view page with Talkback on Android
2. attempt to move through the list (note the jittering and incorrectly moving content)

### Expected Behavior

Swiping left and right should provide consistent navigation without large UI jumps and page jittering. The element that is being focused should be indicated to the user.

### Actual Behavior

Swipping left and right with standard mobile screen reader navigation on long dropdowns results in UI reloading errors and focus issues. Occasionally, the first or last few elements will function normally but issues occur at the midpoints of these lists.

Looking at other UIs that have a similar element the Multiple Search Selection "state" field [here](https://semantic-ui.com/modules/dropdown.html) seems very good, UI focus is maintained and there are no large reloading jumps when navigating. Though this is an arbatray dropdown example, and I am sure there are other ones that could be identified. 

### Environment

- Angular:  20.3.7
- CDK/Material: 20.2.9
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): Android

**Comment by crisbeto** [comment]
I'm not sure if we can do much about it and we don't have a way to properly test Talkback at the moment. It may be worth checking once #32363 lands which will resolve a handful of related a11y issues.
